ui/index.d.ts

```diff
@@ -704,11 +706,8 @@ export type PromptResult = {
 };
 
 /**
- * Wrapper interface of several `vscode.window` methods that handle user input. The main reason for this interface
- * is to facilitate unit testing in non-interactive mode with the `TestUserInput` class.
- * However, the `AzureUserInput` class does have a few minor differences from default vscode behavior:
- * 1. Automatically throws a `UserCancelledError` instead of returning undefined when a user cancels
- * 2. Persists 'recently used' items in quick picks and displays them at the top
```

Comment:
Yes these still apply, but there's a bunch of _other_ differences now. Rather than try to keep a list of all the differences at the top-level here, I thought it would be easier to document that info on the method itself or on the `options` object. For example, the docs for `showQuickPick` already mention 'recently used'